web/src/graphql-operations.ts

```diff
@@ -0,0 +1,239 @@
+import * as Types from './types';
+
+import gql from 'graphql-tag';
+import * as ApolloReactCommon from '@apollo/react-common';
+import * as ApolloReactHooks from '@apollo/react-hooks';
+
+
+export const GetAllTasksDocument = gql`
+    query GetAllTasks {
+  tasks {
+    _id
+    description
+    status
+    assignee {
+      _id
+      name
+      image
+      user_id
+    }
+  }
+}
+    `;
+
+/**
+ * __useGetAllTasksQuery__
+ *
+ * To run a query within a React component, call `useGetAllTasksQuery` and pass it any options that fit your needs.
+ * When your component renders, `useGetAllTasksQuery` returns an object from Apollo Client that contains loading, error, and data properties
+ * you can use to render your UI.
+ *
+ * @param baseOptions options that will be passed into the query, supported options are listed on: https://www.apollographql.com/docs/react/api/react-hooks/#options;
+ *
+ * @example
+ * const { data, loading, error } = useGetAllTasksQuery({
+ *   variables: {
+ *   },
+ * });
+ */
+export function useGetAllTasksQuery(baseOptions?: ApolloReactHooks.QueryHookOptions<Types.GetAllTasksQuery, Types.GetAllTasksQueryVariables>) {
+        return ApolloReactHooks.useQuery<Types.GetAllTasksQuery, Types.GetAllTasksQueryVariables>(GetAllTasksDocument, baseOptions);
+      }
+export function useGetAllTasksLazyQuery(baseOptions?: ApolloReactHooks.LazyQueryHookOptions<Types.GetAllTasksQuery, Types.GetAllTasksQueryVariables>) {
+          return ApolloReactHooks.useLazyQuery<Types.GetAllTasksQuery, Types.GetAllTasksQueryVariables>(GetAllTasksDocument, baseOptions);
+        }
+export type GetAllTasksQueryHookResult = ReturnType<typeof useGetAllTasksQuery>;
+export type GetAllTasksLazyQueryHookResult = ReturnType<typeof useGetAllTasksLazyQuery>;
+export type GetAllTasksQueryResult = ApolloReactCommon.QueryResult<Types.GetAllTasksQuery, Types.GetAllTasksQueryVariables>;
+export const AddTaskDocument = gql`
+    mutation AddTask($task: TaskInsertInput!) {
+  task: insertOneTask(data: $task) {
```

Comment:
Indentation looks a bit off here.

Reply:
Yeah it's wonky but that's just how the generated code comes out. I guess I could add a step that runs prettier on the generated files 🤔

Reply:
Oops, nevermind.